unit_tests/test_sct_events_nodetool.py

```diff
@@ -23,58 +23,58 @@
 class TestNodetoolEvent(unittest.TestCase):
 
     def test_nodetool_cmd_no_options(self):
-        event = NodetoolEvent(type="scrub --skip-corrupted drop_table_during_repair_ks_0",
-                              subtype='start',
-                              severity=Severity.NORMAL,
-                              node='1.0.0.121',
-                              options="")
+        event = NodetoolEvent(nodetool_command="scrub", node='1.0.0.121',
+                              options="", publish_event=False)
```

Comment:
I doesn't call. We can know it because of the test doesn't fail. In case it is called, we will receive exception:
```
RuntimeError: You should create default EventsProcessRegistry first
```

Reply:
It is white-box knowledge which can be changed. I refered to covering interface and checking it, such approach doesn't depend on the white-box knowledge. So, concern is still valid. Need to add small assertions using "mock" module.

Reply:
Added test to `unit_tests/test_events.py`